app/assets/stylesheets/application.scss

```diff
@@ -12,3 +12,5 @@
 @import "devise/index";
 @import "pages/index";
 @import "components/index";
+@import "components/buttons";
+@import "components/cards"
```

Comment:
We don't import components in the application.scss
We import them in the components/_index.scss and this file is already imported in the application.Scss